prow/scripts/skr-integration-own-cluster.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script is designed to run SKR integration test with own_cluster plan.
+
+set -o errexit
+
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly TEST_INFRA_SOURCES_DIR="$(cd "${SCRIPT_DIR}/../../" && pwd)"
+
+# shellcheck source=prow/scripts/lib/log.sh
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/log.sh"
+# shellcheck source=prow/scripts/lib/utils.sh
+source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/utils.sh"
+
+if [[ "${BUILD_TYPE}" == "pr" ]]; then
+  log::info "Execute Job Guard"
+  "${TEST_INFRA_SOURCES_DIR}/development/jobguard/scripts/run.sh"
+fi
+
+export SHOOT_DEFINITION_PATH="${TEST_INFRA_SOURCES_DIR}/prow/scripts/resources/skr-test/shoot-template.yaml"
+export GARDENER_SHOOT_TEMPLATE="$(envsubst < $SHOOT_DEFINITION_PATH | base64)"
```

Comment:
looks like linter is not happy about quoting. I think it expects doublequoted parameter expansion instead of the whole command

```
In ./prow/scripts/skr-integration-own-cluster.sh line 21:
export GARDENER_SHOOT_TEMPLATE="$(envsubst < $SHOOT_DEFINITION_PATH | base64)"
                                             ^-- SC2086: Double quote to prevent globbing and word splitting.
```

```suggestion
export GARDENER_SHOOT_TEMPLATE=$(envsubst < "$SHOOT_DEFINITION_PATH" | base64)
```